tests/blackbox/stratis_cli_cert.py

```diff
@@ -49,16 +77,219 @@ def setUp(self):
         StratisCli.destroy_all()
         assert StratisCli.pool_list() == []
 
+    def unittest_command(  # pylint: disable=bad-continuation
+        self, args, exp_exit_code, exp_stderr_is_empty, exp_stdout_is_empty
+    ):
+        """
+        Execute a test command and make assertions about the exit code, stderr, and stdout
+        :param self: A test object
```

Comment:
Please leave out the doc line for ```self```. It's never documented if the method is part of a class.